src/lib/isColor.js

```diff
@@ -0,0 +1,20 @@
+import assertString from './util/assertString';
+
+const colorFormats = {
```

Comment:
These three are already checked in separate validators, why not use those instead of re-defining regexes?

Reply:
I primarily believed that a common use case for an isColor validator would be (from personal experience, since i needed a color validator in my own application) color format that would work on the browser. Currently existing isHexColor and isRgbColor fail validation for commonly used color format cases such as a space for separating rgb values: rgb(255, 255, 255) (would fail even though it works and is commonly used in css) or succeed validation for unprefixed hex colors (which do not work with css). Hence I thought about creating my own regexes for these working commonly used edge cases (and some not so commonly used).

Although I do see your point, in that isHSL has regex better created than what i created and it would be better to use existing color validators and use isColor as an integration validator for these individual formats. I will replace my regex with existing ones and commit asap.

Thanks for you time! 😀